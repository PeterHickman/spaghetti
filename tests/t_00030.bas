10 REM Check that the whitepace in strings is not being eaten
20 PRINT "The next two strings should be the same length"
30 PRINT
40 PRINT "start                             end"
50 PRINT "start --------------------------- end"
60 END
