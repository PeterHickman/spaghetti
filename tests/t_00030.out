The next two strings should be the same length

start                             end
start --------------------------- end
